examples/fields.go

```diff
@@ -3,16 +3,17 @@ package main
 
 import (
 	"fmt"
+	"os"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/client/v1/nimbleos"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/param"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/service"
 )
 
 func main() {
 	groupService, err := service.NewNimbleGroupService(
-		service.WithHost("1.1.1.1"),
-		service.WithUser("xxx"),
-		service.WithPassword("xxx"))
+		service.WithHost(os.Getenv("GO_SDK_TEST_HOST")),
+		service.WithUser(os.Getenv("GO_SDK_TEST_USER")),
```

Comment:
I wonder if "GO" is important to include in the env var names, or even "SDK".  It is tempting to use "NIMBLE", like "NIMBLE_TEST_USER", but we might be better off avoiding the branding even though that term is all over the SDK.

Also, these are examples rather than tests, so maybe TEST isn't right either.

How about SDK_TARGET_HOST, SDK_TARGET_USER, SDK_TARGET_PASSWORD?